cylc/flow/scripts/validate_reinstall.py

```diff
@@ -135,10 +135,16 @@ def vro_cli(parser: COP, options: 'Values', workflow_id: str):
     try:
         detect_old_contact_file(workflow_id)
     except ServiceFileError:
-        # Workflow is definately running:
+        # Workflow is definitely running:
         workflow_running = True
+    except Exception:
+        LOG.critical(
+            'Cannot tell if the workflow is running'
+            '\nNote, Cylc 8 cannot restart Cylc 7 workflows.'
+        )
```

Comment:
After discussion in another pr...
https://github.com/cylc/cylc-flow/pull/5323#issuecomment-1403731534

This error message needs to be raised on other commands when interacting with running cylc 7 workflow.

Reply:
From my investigation it is only `cylc play` which doesn't raise this error.

Reply:
I don't think this is resolved? `cylc play` needs to inform users of this message when playing a running Cylc 7 workflow. I checked the other commands and they fail correctly.